proto/sources.proto

```diff
@@ -270,3 +271,14 @@ message SlackRealtime {
     credentials.SlackTokens tokens = 1;
   }
 }
+
+message Sharepoint {
+  string endpoint = 1 [(validate.rules).string.uri_ref = true];
+  oneof credential {
+    string token = 2;
+    credentials.ClientCredentials authenticated = 3;
```

Comment:
remove this, and remove ClientCredentials from the credentials proto because it won't be used

Reply:
@ahrav it looks like you are using ClientCredentials as well - are we good to remove? cc: @dustin-decker 

https://github.com/trufflesecurity/trufflehog/blob/b3d917f9c71789f67e71bed6c3e106a6cbd6d012/proto/sources.proto#L260

Reply:
Okay, leave it if it's being used elsewhere